Formula/argyll-cms.rb

```diff
@@ -14,7 +14,9 @@ class ArgyllCms < Formula
 
   depends_on "jam" => :build
   depends_on "jpeg"
+  depends_on "libpng"
   depends_on "libtiff"
+  depends_on "zlib"
```

Comment:
You can get it to use the system version by doing
```
inreplace "Jamtop", "GLOB /usr/lib : lib$(lcase).so", "GLOB /usr/lib : lib$(lcase).dylib"
```
which would need to be reported upstream since they need to be looking for .dylib not .so on macOS. But it's fine to just let it use its vendored version of zlib.

Reply:
Will report upstream, hopefully they'll get this resolved!